src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller\Renaissance\DepartmentSite;
+
+use App\Entity\DepartmentSite\DepartmentSite;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Entity;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/federations", name="app_renaissance_department_site_")
+ */
+class DepartmentSiteController extends AbstractController
+{
+    /**
+     * @Route("/{slug}", name="view"), methods={"GET"}
+     * @Entity("departmentSite", expr="repository.findOneBySlug(slug)")
+     */
+    public function departmentSiteAction(DepartmentSite $departmentSite): Response
```

Comment:
what about making this controller invokable, with `__invoke` method?

Reply:
I will add the list action in this file on another PR